lib/rubocop/cop/mavenlint/use_application_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Mavenlint
+      # Enforce models inheriting from ApplicationController instead of ActionController::Base
+      #
+      # For example
+      #
+      #   class SomeController < ActionController::Base
+      #   end
+      #
+      # should be
+      #
+      #   class SomeController < ApplicationController
+      #   end
+      #
+      class UseApplicationController < RuboCop::Cop::Cop
+        MSG = 'Controllers should subclass `ApplicationController`.'.freeze
+        SUPERCLASS = 'ApplicationController'.freeze
+        BASE_PATTERN = '(const (const nil? :ApplicationController) :Base)'.freeze
```

Comment:
Should this be:

```lisp
(const (const nil? :ActionController) :Base)
```

?

A test could help give us confidence

Reply:
Thanks for always keeping us on track! I added some tests around it, and fixed the mistakes 😄

Reply:
@ahuth  It looks like you are the owner on ruby gems, could you add me? 
`gem owner mavenlint --add brent.w.gaither@gmail.com`

Reply:
You're now an owner!